data.json

```diff
@@ -1,1490 +1 @@
-[
```

Comment:
Please dont remove the data.json file

Reply:
It's not removed, it is actually sorted, but for some reason it turns out to be a minified one, working on unminifying it